.github/workflows/bot.yml

```diff
@@ -33,15 +33,18 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: "18.14.1"
-          
+
       - name: Install
         run: yarn install
-        
+
       - name: Authenticate as UbiquiBot
         uses: tibdex/github-app-token@v1.7.0
         id: get_installation_token
         with:
           app_id: 346074
+          # https://github.com/organizations/ubiquibot/settings/apps/ubiquibot-qa/installations
+          # https://github.com/apps/ubiquibot-qa/installations/new/permissions?target_id=4975670
+          # https://github.com/settings/installations/38790399
```

Comment:
I am unsure which is the correct link but this is to install the app instance on a fork so that this authenticates. These are the three links from the whole install flow.